ibrahim/styles.scss

```diff
@@ -0,0 +1,265 @@
+* {
+  box-sizing: border-box;
+}
+
+/* Undoing defaults */
+body,
+.body {
```

Comment:
I would generally avoid a classname of "body" -- since it matches an HTML element, it's too easy for the class to look like a typo.  See my notes about the HTML for suggestions about this.